pkg/runtime/reconciler.go

```diff
@@ -106,14 +107,20 @@ func (r *reconciler) reconcile(req ctrlrt.Request) error {
 
 	acctID := r.getOwnerAccountID(res)
 	region := r.getRegion(res)
+	roleARN, _ := r.cache.Accounts.GetAccountRoleARN(string(acctID))
+	sess, err := NewSession(region, ackv1alpha1.AWSResourceName(roleARN))
+	if err != nil {
+		return err
+	}
 
 	r.log.WithValues(
 		"account_id", acctID,
+		"assumed_role_arn", roleARN,
```

Comment:
just call it `role` instead of `assumed_role_arn`. And let's call the `account_id`log message key just `account`.

Reply:
Done!